### PR TITLE
[NUI] Rename some properties of WebView, WebSettings, etc.

### PR DESCRIPTION
**Pull request by huayongxu**
This patch is to raname some properties to follow naming rule.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by zg2nets — COMMENTED** [review]
plz fix build error

**Comment by huayongxu** [comment]
> plz fix build error

Done.

File: src/Tizen.NUI/src/internal/WebView/WebContext.cs [hunk 1]
```diff
@@ -168,11 +168,11 @@ public bool DisableCache
         {
             get
             {
-                return disableCache;
+                return cacheDisabled;
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView/WebContext.cs:171]: Shouldn't we have to change property name too?

Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebContext.cs:171]: this property would be changed by another patch [1].
after [1] merged, I will update current patch.

[1] https://github.com/Samsung/TizenFX/pull/2940.

Comment (rabbitfor) [reply src/Tizen.NUI/src/internal/WebView/WebContext.cs:171]: OK. I merged #2940 just now. It uses `CachedEnabled`. Please update the patch.

Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebContext.cs]: Done.

File: src/Tizen.NUI/src/internal/WebView/WebSettings.cs [hunk 2]
```diff
@@ -403,28 +403,28 @@ public bool ImePanelEnabled
...
             }
         }
 
         /// <summary>
-        /// Allow Images Load Automatically.
+        /// Allow images load automatically or not.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool AllowImagesLoadAutomatically
+        public bool ImagesLoadAutomaticallyAllowed
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView/WebSettings.cs:427]: AutomaticImageLoadingAllowed?

Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebSettings.cs]: Done.

**Review by rabbitfor — APPROVED** [review]
LGTM

**Pull request merged**